xarray/core/common.py

```diff
@@ -346,6 +346,28 @@ def assign_coords(self, **kwargs):
         data.coords.update(results)
         return data
 
+    def assign_attrs(self, *args, **kwargs):
+        """Assign new attrs to this object, returning a new object equivalent
```

Comment:
The second version is preferred. But you're right that `assign_coords` also has a docstring that violates this.